crates/turbopack-ecmascript/src/transform/mod.rs

```diff
@@ -33,6 +33,10 @@ pub enum EcmascriptInputTransform {
     React {
         #[serde(default)]
         refresh: bool,
+        // swc.jsc.transform.react.importSource
+        import_source: OptionStringVc,
+        // swc.jsc.transform.react.runtime,
+        runtime: OptionStringVc,
```

Comment:
I would rather this be an enum instead, since there are only two valid values. This avoids downstream validation.